### PR TITLE
Close up file connections in CommCareConfigEngine

### PR DESCRIPTION
**Pull request by wpride**

### CODE REVIEW

File: src/cli/java/org/commcare/util/engine/CommCareConfigEngine.java [hunk 1]
```diff
@@ -149,12 +149,27 @@ private String downloadToTemp(String resource) {
...
+                try {
+                    if (fos != null) {
+                        fos.close();
+                    }
+                    if (bis != null) {
+                        bis.close();
+                    }
+                    conn.disconnect();
+                } catch (IOException ex) {
+                    // Log error writing file and bail out.
```
Comment (amstone326) [inline src/cli/java/org/commcare/util/engine/CommCareConfigEngine.java]: @wpride Did you want to actually log something here?

**Comment by amstone326** [comment]
@wpride Do you still want this to get in? Looks like it's been sitting for a long while

**Comment by amstone326** [comment]
@wpride Bumping my question above ^^

**Comment by wpride** [comment]
@amstone326 good to go

**Pull request merged**